src/ensenso.cpp

```diff
@@ -969,4 +969,40 @@ Result<Ensenso::CaptureParams> Ensenso::getCaptureParameters(bool crop_to_roi) {
 	return params;
 }
 
+bool Ensenso::isDumpingTree() {
+	return dump_tree_;
+}
+
+void Ensenso::enableNxLibLogging(std::string connection_folder_path, std::string debug_level, int log_file_size, bool dump_tree) {
+	NxLibItem debugOut = root[itmDebug][itmFileOutput];
```

Comment:
```suggestion
	NxLibItem debug = root[itmDebug][itmFileOutput];
```

Or alternatively `debug_out`, because we don't use lowerCamelCase for variables.